src/modules/CaaSMapper.spec.ts

```diff
@@ -1009,36 +1143,80 @@ describe('CaaSMapper', () => {
         },
         children: [],
       })
-      expect(mapper.mapDataEntries).toHaveBeenCalledWith(section.formData, [...path, 'data'])
+      expect(mapper.mapDataEntries).toHaveBeenCalledWith(section.formData, [
+        ...path,
+        'data',
+      ])
+    })
+
+    it.only('should show or not displayed property in section', async () => {
```

Comment:
it.only?

Also please fix the syntax/grammar in the test name. It's very difficult to understand from the name what the test is supposed to do.